lib/api/tests/instance.rs

```diff
@@ -0,0 +1,89 @@
+use anyhow::Result;
+#[cfg(feature = "js")]
+use wasm_bindgen_test::*;
+
+use wasmer::*;
+
+
+#[cfg(feature = "js")]
+#[cfg_attr(feature = "js", wasm_bindgen_test)]
+fn exports_work_after_multiple_instances_have_been_freed_js() {
+    exports_work_after_multiple_instances_have_been_freed().unwrap();
+}
+
+#[cfg_attr(feature = "sys", test)]
+fn exports_work_after_multiple_instances_have_been_freed() -> Result<()> {
```

Comment:
Could we do this with a macro?

```rust
#[universal_test]
fn exports_work_after_multiple_instances_have_been_freed() -> Result<()> {
```

The macro could expand into something like:
```rust
#[wasm_bindgen_test]
fn exports_work_after_multiple_instances_have_been_freed_js() {
    fn exports_work_after_multiple_instances_have_been_free().unwrap();
}

#[test]
fn exports_work_after_multiple_instances_have_been_freed() -> Result<()> {
    // ...
}
```

That way the code is not as repetitive